pset-iterators.js

```diff
@@ -128,6 +143,8 @@ declared with const
 console.log('Problem 4:')
 
 // Add your code below this line
+const livingPresidents = presidents.map(president =>
```

Comment:
@jimernest90 everything is structured correctly, however `.filter()` was the iterator to use here instead of `.map()` - see [this approach](https://github.com/jsd20190730/hw-02-functions-and-iterators-solution/blob/master/pset-iterators.js#L152)